prospero-cli/src/test/java/org/wildfly/prospero/cli/RepositoryDefinitionTest.java

```diff
@@ -75,5 +92,48 @@ public void throwsErrorIfFormatIsIncorrect() throws Exception {
         assertThrows(ArgumentParsingException.class, ()->from(List.of("foo::bar::http://test1.te")));
 
         assertThrows(ArgumentParsingException.class, ()->from(List.of("imnoturl")));
+
+    }
+
+    @Test
+    public void throwsErrorIfFormatIsIncorrectForFileURLorPathDoesNotExist() throws Exception {
+        assertThrows(ArgumentParsingException.class, ()->from(List.of("::"+fileURL)));
+
+        assertThrows(ArgumentParsingException.class, ()->from(List.of("repo-1::")));
+
+        assertThrows(ArgumentParsingException.class, ()->from(List.of("repo-1:::"+fileURL)));
+
+        assertThrows(ArgumentParsingException.class, ()->from(List.of("foo::bar::"+fileURL)));
+
+        assertThrows(ArgumentParsingException.class, ()->from(List.of("file:/path/to/repo")));
+
+        assertThrows(ArgumentParsingException.class, ()->from(List.of(filePath)));
+
+        assertThrows(ArgumentParsingException.class, ()->from(List.of("file://../repo")));
+
+    }
+
+    @Test
+    public void testCorrectRelativeOrAbsolutePathForFileURL() throws Exception {
```

Comment:
what about a relative path?

Reply:
The relative path is test in the first lines of this method with this url "file:../prospero-common"

Reply:
That's a URI with relative path. A relative path would be `../prospero-common` on its own

Reply:
I believe this verifies JBEAP-26295 as well so those are valid tests. Just adding some examples of paths will be enough.